packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -109,3 +152,111 @@ Future<Set<String>> readDepfile(String depfilePath) async {
       .where((String path) => path.isNotEmpty)
       .toSet();
 }
+
+/// Dart snapshot builder.
+///
+/// Builds Dart snapshots in one of three modes:
+///   * Script snapshot: architecture-independent snapshot of a Dart script core
```

Comment:
this seems to switch between singular and plural mid-sentence?

Reply:
Missed a word! Done.